pkg/distro/fedora/distro.go

```diff
@@ -745,6 +750,13 @@ func newDistro(version int) distro.Distro {
 		minimalrawImgType,
 	)
 
-	rd.addArches(x86_64, aarch64)
+	ppc64le.addImageTypes(
+		&platform.PPC64LE{
+			BIOS: true,
+		},
+		liveInstallerImgType,
```

Comment:
I'm wondering if we should enable the ppc64le architecture also for other image types, mainly the `qcow2` type. The COPR team has been asking for this for ages... :thinking: